geotrek/api/tests/test_v2.py

```diff
@@ -341,7 +341,10 @@ def setUpTestData(cls):
             reservation_system=cls.reservation_system,
             practice=cls.practice,
             difficulty=cls.difficulty,
-            accessibility_level=cls.accessibility_level
+            accessibility_level=cls.accessibility_level,
+            description='<p>Description</p>'
+                        '<img src="/media/upload/steep_descent.svg" alt="Descent">'
+                        '<img src="https://rando-millevaches-admin.fr/media/upload/pedestre.svg" alt="" width="1848" height="1848">'
```

Comment:
```suggestion
                        '<img src="https://rando-millevaches-admin.fr/media/upload/pedestre.svg" alt="" width="1848" height="1848">'
```
don't let real URL in public github commit